CONTRIBUTING.md

```diff
@@ -20,18 +20,18 @@ The following table explains how we interpret each of the states for a Pivotal s
 
 ## Git Workflow
 
-- We use [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/).
+- We use [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)
```

Comment:
It is git-flow tho? We don't have a `development` branch and do PRs directly to main. 

I do think we should have a dev branch

Reply:
fair!!